projects/kit/components/line-clamp/line-clamp.component.ts

```diff
@@ -8,13 +8,26 @@ import {
     Inject,
     Input,
     NgZone,
+    Output,
     Renderer2,
     ViewChild,
 } from '@angular/core';
 import {isCurrentTarget, tuiDefaultProp, tuiPure, typedFromEvent} from '@taiga-ui/cdk';
+import {
+    TUI_LINE_CLAMP_OPTIONS,
+    TuiLineClampOptions,
+} from '@taiga-ui/kit/components/line-clamp/line-clamp-options';
```

Comment:
```suggestion
} from './line-clamp-options';
```
Otherwise, build will be broken

Reply:
Sorry, didn't notice that, I'll fix it now